composer.json

```diff
@@ -42,7 +42,7 @@
         "api-platform/core": "^2.5",
         "auth0/auth0-php": "^5.4 || ^7.6",
         "bugsnag/bugsnag-laravel": "^2.17",
-        "doctrine/dbal": "^2.6",
+        "doctrine/dbal": "^2.11",
```

Comment:
I believe we shouldn't do it because it will impact consumers of the package. Let's take care about method existence check instead.

Reply:
Fixed